exchanges/alphapoint/alphapoint_wrapper.go

```diff
@@ -216,3 +216,117 @@ func (a *Alphapoint) GetFeeByType(feeBuilder exchange.FeeBuilder) (float64, erro
 func (a *Alphapoint) GetWithdrawCapabilities() uint32 {
 	return a.GetWithdrawPermissions()
 }
+
+// GetActiveOrders retrieves any orders that are active/open
+func (a *Alphapoint) GetActiveOrders(getOrdersRequest exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	resp, err := a.GetOrders()
+	if err != nil {
+		return nil, err
+	}
+
+	var orders []exchange.OrderDetail
+	for x := range resp {
+		for _, order := range resp[x].OpenOrders {
+			if order.State != 1 {
+				continue
+			}
+
+			orderDetail := exchange.OrderDetail{
+				Amount:          float64(order.QtyTotal),
+				OrderDate:       order.ReceiveTime,
+				Exchange:        fmt.Sprintf("%v - %v", a.Name, order.AccountID),
+				ID:              fmt.Sprintf("%v", order.ServerOrderID),
+				Price:           float64(order.Price),
+				RemainingAmount: float64(order.QtyRemaining),
+			}
+
+			if order.Side == 1 {
+				orderDetail.OrderSide = string(exchange.BuyOrderSide)
+			} else if order.Side == 2 {
+				orderDetail.OrderSide = string(exchange.SellOrderSide)
+			}
+
+			switch order.OrderType {
+			case 1:
+				orderDetail.OrderType = string(exchange.MarketOrderType)
+			case 2:
+				orderDetail.OrderType = string(exchange.LimitOrderType)
+			case 3:
+				fallthrough
```

Comment:
Is there ever going to be a case where this is used?
Go supports multiple comma separated values on a switch 
Is it worth grouping 3,4 & 5,6? 

Might even be worth moving this over to a map there is cases where a map lookup on small data set is faster than a switch obviously some benchmarking might be needed to see if its even worth it and not premature optimization

Could also be used to remove some code duplication
https://github.com/thrasher-/gocryptotrader/blob/obtain-older-orders/exchanges/exchange.go#L826
Could be replaced with a map like

```
orderType := map[int]string{
		0: "ANY",
		1: "LIMIT",
		2: "MARKET",
		3: "IMMEDIATE_OR_CANCEL",
		4: "STOP",
		5: "TRAILINGSTOP",
		6: "UNKNOWN",
	}
```
and remove the additional switch and just do a lookup of the map
```
orderDetail.OrderType = orderType[order.OrderType]
```

Would most likely want to add some error checking to it to make sure the value is found/valid
A map lookup will return value, value found (boolean)

Reply:
I didn't know about the switch commas, thanks!
Order types are already used when submitting an order. While not all exchanges return the type of order in this PR, 12 do, so its worth keeping the data if its returned.

I'll look at using a map for it all now